ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-commands.sh

```diff
@@ -50,7 +45,7 @@ networking:
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OpenShiftSDN
+  networkType: OVNKubernetes
```

Comment:
In 4.11, SDN is the default, while in 4.12+, OVN is the default.
We may need two conf to cover both situations, or we will get into trouble when we backport the tests for old OCP versions.

Reply:
@sgaoshang maybe we can delete the `networkType` setting here.

Reply:
Agree we can set networkType in other PR

Reply:
I meant, for this step, the whole `networkType:` line should be deleted now, let installer to choose the default network type per the selected payload.

On other platform, the similar steps - ipi-conf-xxxx, never set an explicit `networkType`

Reply:
@sgaoshang can we also delete `networkType:` line together in this PR?